### PR TITLE
Fix coord system and add non-human test data

### PR DESCRIPTION
**Pull request by dglemos**
HGVS genomic didn't work for some non-human species because they only have `primary_assembly` coord system.
Linked to https://github.com/Ensembl/ensembl-variation/pull/592 
Example for cow:
1:g.346786A>C

### CODE REVIEW

**Review by helensch — CHANGES_REQUESTED** [review]
The code update works on release databases.

Modifications are need for the test to use the new test database.
The new test databases (variation and core) should include a table.sql to be set up.

I suggest also testing that the new test database can be patched for future releases

File: modules/t/variationFeatureAdaptor.t [hunk 1]
```diff
@@ -516,4 +516,25 @@ throws_ok {$vfa->fetch_by_hgvs_notation('ENST00000470094.1:c.59A>G'); }qr/Refere
 my $ok_hgvs = 'ENSP00000293261.2:p.Ser455del';
 $vf = $vfa->fetch_by_hgvs_notation($ok_hgvs);
 ok($vf->allele_string eq 'AGC/-', "HGVSp matches reference");
+
+my $lrg_hgvsg = 'LRG_293:g.69535N>G';
```
[the remark this answers is not in the document]
Comment (dglemos) [reply modules/t/variationFeatureAdaptor.t:520]: As the branching is happening now, we'll leave the tests as they are now. 
The reference allele will be updated later as described in ENSVAR-3862.

**Comment by dglemos** [comment]
@helensch thank you for the review. As discussed, the test db will be updated later as described here ENSVAR-3862.

**Review by helensch — APPROVED** [review]

**Comment by helensch** [comment]
Merged into master and release/103

**Pull request merged**